tslint.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "@osu-cass/tslint-config",
   "rules": {
-    "strict-boolean-expressions": [true, "allow-string", "allow-number", "allow-boolean-or-undefined", "allow-mix"]
+    "strict-boolean-expressions": false
```

Comment:
You don't need to do this. Update `@osu-cass/tslint-config` to the latest version, and you'll get this in the configuration without having to override.